packages/fluentui/react-bindings/src/hooks/useIsomorphicLayoutEffect.ts

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 
 // useLayoutEffect() produces a warning with SSR rendering
 // https://medium.com/@alexandereardon/uselayouteffect-and-ssr-192986cdcf7a
-const useIsomorphicLayoutEffect = typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
+const useIsomorphicLayoutEffect =
+  typeof window !== 'undefined' && process.env.NODE_ENV !== 'test' ? React.useLayoutEffect : React.useEffect;
```

Comment:
Without `process.env.NODE_ENV !== 'test'` it produces warning for tests because we use JSDom which has `window` for sure